### PR TITLE
Adds Planetary's Donor items, and some adminbus stuff

### PR DESCRIPTION
**Pull request by Stingray540**
:cl: Stingray
rscadd: Added Planetary's donor armour
rscadd: Added Silent Shadow for adminbus purposes. Sprites provided by ComradeSocks
/:cl:

### CODE REVIEW

File: code/modules/halo/covenant/species/sangheili/clothing.dm [hunk 1]
```diff
@@ -376,6 +376,49 @@
...
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|BLOCKHAIR
+	body_parts_covered = HEAD|FACE
+	cold_protection = HEAD
+	min_cold_protection_temperature = SPACE_HELMET_MIN_COLD_PROTECTION_TEMPERATURE
+
+/obj/item/clothing/suit/armor/special/combatharness/silentshadow
+	name = "Sangheili Combat Harness (Silent Shadow)"
+	icon_state = "ss_chest_obj"
+	item_state = "ss_chest"
+	slowdown_general = -2 //They're meant to be primarily melee-only and admemespawn anyway, so.
```
Comment (X0-11) [inline code/modules/halo/covenant/species/sangheili/clothing.dm:397]: dear lord do you understand how harsh this is of a slowdown

File: code/modules/halo/covenant/species/sangheili/clothing.dm [hunk 2]
```diff
@@ -376,6 +376,49 @@
...
+	totalshields = 210
+	specials = list(/datum/armourspecials/shields/elite,/datum/armourspecials/shieldmonitor/sangheili,/datum/armourspecials/cloaking/silentshadow)
+	action_button_name = "Toggle Active Camouflage"
+	item_flags = STOPPRESSUREDAMAGE | THICKMATERIAL | AIRTIGHT
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	cold_protection = UPPER_TORSO|LOWER_TORSO|ARMS
+	min_cold_protection_temperature = SPACE_SUIT_MIN_COLD_PROTECTION_TEMPERATURE
```
Comment (X0-11) [inline code/modules/halo/covenant/species/sangheili/clothing.dm:404]: is this meant to be a by-default space capable suit

File: code/modules/halo/misc/armourspecials/cloaking.dm [hunk 3]
```diff
@@ -65,3 +65,8 @@
 			disrupt_cloak(cloak_recover_time*2)
 		if(2)
 			disrupt_cloak(cloak_recover_time*4)
+
+/datum/armourspecials/cloaking/silentshadow
+	min_alpha=1 ///These are meant to be 100% admin-spawn kill-squads, so... have fun
```
Comment (X0-11) [inline code/modules/halo/misc/armourspecials/cloaking.dm:70]: fear

**Review by X0-11 — CHANGES_REQUESTED** [review]

File: code/modules/halo/covenant/species/sangheili/clothing.dm (hunk 2 above)
Comment (Stingray540) [reply code/modules/halo/covenant/species/sangheili/clothing.dm:404]: yes.

File: code/modules/halo/covenant/species/sangheili/clothing.dm (hunk 1 above)
Comment (Stingray540) [reply code/modules/halo/covenant/species/sangheili/clothing.dm:397]: It's a speed UP, isn't it?

File: code/modules/halo/misc/armourspecials/cloaking.dm (hunk 3 above)
Comment (Stingray540) [reply code/modules/halo/misc/armourspecials/cloaking.dm:70]: Yep, true invisibility. Intended soley for adminbus.

File: code/modules/halo/covenant/species/sangheili/clothing.dm (hunk 1 above)
Comment (X0-11) [reply code/modules/halo/covenant/species/sangheili/clothing.dm:397]: true, same point applies but replace harsh with powerful and slowdown with speedup

Comment (Stingray540) [reply code/modules/halo/covenant/species/sangheili/clothing.dm:397]: This was what the old Zeal suit was and that was... manageable. Isn't T-voan something like -4 as well? Again, these are meant to be deathsquad-level bullshit that will probably only by spawned as counter for UNSC adminbus bullshit. If these ever make it into actual rotation then yes, remove all the additions.

Comment (X0-11) [reply code/modules/halo/covenant/species/sangheili/clothing.dm:397]: yeah but even as adminbus this is way too fast and is going to make them just something players dispise seeing

Comment (Stingray540) [reply code/modules/halo/covenant/species/sangheili/clothing.dm:397]: I really don't think it's that bad from my (admittedly brief) fuckery with them. They're mostly meant to be melee only memes as well. But if this is your hard line I'll change it, I just don't see why. I can just varedit them to be like this anyway?

Comment (X0-11) [reply code/modules/halo/covenant/species/sangheili/clothing.dm:397]: mm nah it's fine, i'll let 'em through if they're meant for melee shenanigans

**Review by X0-11 — APPROVED** [review]

**Pull request merged**